### PR TITLE
support Scala 2.13.0. add scripted test

### PR DESCRIPTION
**Pull request by xuwei-k**

### CODE REVIEW

File: src/main/scala/com/softwaremill/SbtSoftwareMill.scala [hunk 1]
```diff
@@ -139,8 +145,17 @@ object SbtSoftwareMill extends AutoPlugin {
...
       ),
-      addCompilerPlugin("org.spire-math"  %% "kind-projector" % "0.9.10"),
-      addCompilerPlugin("org.scalamacros" % "paradise"        % "2.1.1" cross CrossVersion.patch),
+      addCompilerPlugin("org.typelevel" %% "kind-projector" % "0.10.0"),
+      libraryDependencies ++= {
+        CrossVersion.partialVersion(scalaVersion.value) match {
+          case Some((2, v)) if v <= 12 =>
+            Seq(compilerPlugin("org.scalamacros" % "paradise" % "2.1.1" cross CrossVersion.patch))
+          case _ =>
+            // if scala 2.13.0-M4 or later, macro annotations merged into scala-reflect
```
Comment (kubukoz) [inline src/main/scala/com/softwaremill/SbtSoftwareMill.scala]: IIRC to enable annotations you still need a compiler flag

**Comment by adamw** [comment]
Thanks! For some reason I wasn't watching this repository and didn't notice the PR. Released 1.7.0

**Comment by kubukoz** [comment]
@adamw I think the compiler flag for macro annotations is still missing in the 2.13 setup (makes sense to enable it since the plug-in is enabled on other versions).

**Comment by adamw** [comment]
@kubukoz but there was no compiler flag in the original, just the dependency? So I suppose this change shouldn't break anything

**Comment by adamw** [comment]
but we'll see and adjust ;)

**Comment by kubukoz** [comment]
I mean, the dependency was enough on < 2.13, now there's a flag instead. I suppose it would be the same behavior if we let the flag in.

**Comment by adamw** [comment]
Ah :) what's the flag?

**Comment by adamw** [comment]
Found it, it's `-Ymacro-annotations` :)

**Pull request merged**